index.html

```diff
@@ -9,38 +9,223 @@
+        </div>
+        <div class="work-description align-item">
+          <p class="company">CANOPY</p>
+          <div class="separator"></div>
+          <p class="position">Back End Dev</p>
+          <div class="separator"></div>
+          <p class="year">2015</p>
+        </div>
+        <div class="project-description align-item">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="technologies align-item">
+          <div class="technologie-label">
+            <p>html</p>
+          </div>
+          <div class="technologie-label">
+            <p>css</p>
+          </div>
+          <div class="technologie-label">
+            <p>javascript</p>
+          </div>
+        </div>
+        <button class="see-project align-item">
+          See Project
+        </button>
+      </div>
+      <div class="work-container">
+        <img class="align-item" src="./assets/images/work-2.png" alt="Mult-Post Stories">
+        <div class="work-info-container">
+          <h1 class="align-item">Multi-Post Stories</h1>
+        </div>
+        <div class="work-description align-item">
+          <p class="company">CANOPY</p>
+          <div class="separator"></div>
+          <p class="position">Back End Dev</p>
+          <div class="separator"></div>
+          <p class="year">2015</p>
+        </div>
+        <div class="project-description align-item">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="technologies align-item">
+          <div class="technologie-label">
+            <p>html</p>
+          </div>
+          <div class="technologie-label">
+            <p>css</p>
+          </div>
+          <div class="technologie-label">
+            <p>javascript</p>
+          </div>
+        </div>
+        <button class="see-project align-item">
+          See Project
+        </button>
+      </div>
+      <div class="work-container">
+        <img class="align-item" src="./assets/images/work-3.png" alt="Tonic">
+        <div class="work-info-container">
+          <h1 class="align-item">Tonic</h1>
+        </div>
+        <div class="work-description align-item">
+          <p class="company">CANOPY</p>
+          <div class="separator"></div>
+          <p class="position">Back End Dev</p>
+          <div class="separator"></div>
+          <p class="year">2015</p>
+        </div>
+        <div class="project-description align-item">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="technologies align-item">
+          <div class="technologie-label">
+            <p>html</p>
+          </div>
+          <div class="technologie-label">
+            <p>css</p>
+          </div>
+          <div class="technologie-label">
+            <p>javascript</p>
+          </div>
```

Comment:
- [ ] Kindly make sure your tech stack list is grabbed inside a list of element.
`<ul>`
`<li>html</li>`
`<li>css</li>`
`</ul>`

Reply:
Thanks for feedback, Tech skills will be wrapped in a ul and li tag